src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -1432,11 +1410,10 @@ end
 import { Client, Databases, ID } from "https://deno.land/x/appwrite/mod.ts";
                 
 export default function ({req, res, error}: any){
-    const client = new Client()
-        .setEndpoint("https://cloud.appwrite.io/v1")
-        .setProject(Deno.env.get("APPWRITE_FUNCTION_PROJECT_ID") || "")
-        .setKey(Deno.env.get("APPWRITE_API_KEY") || "");
-    
+    const client = new Client();
+    // Set API key
+    client.setKey(req.headers.get('x-appwrite-key') || ""); // API key is automatically provided by Appwrite
```

Comment:
we do `req.headers['x-appwrite-key']` elsewhere so let's use that syntax consistently.

```suggestion
    client.setKey(req.headers['x-appwrite-key'] || ""); // API key is automatically provided by Appwrite
```